src/builder.cpp

```diff
@@ -447,6 +448,10 @@ MeshInstance3D* Builder::build_entity_mesh(int idx, LMEntity& ent, Node3D* paren
 		// Create material
 		Ref<Material> material;
 
+		if (String(tex.name) == m_loader->get_skip_texture_name()) {
```

Comment:
It would probably be good to have a comment here explaining what this is doing (even if it's clear by looking at the code)

Reply:
Additionally, casting `tex.name` to a `String` might invoke a memory allocation & string copy for each texture, we can probably avoid that? (Doesn't `String` have a compare method against `const char*`?)